gui/js/tray.js

```diff
@@ -149,13 +115,57 @@ const init = (app, listener) => {
     cm.insert(1, new MenuItem({ type: 'separator' }))
     tray.setContextMenu(cm)
   }
+
   setStatus('idle')
 }
 
 const wasInitiated = () => {
   return tray != null
 }
 
+// old tray menu
```

Comment:
Why did you keep this commented code?

Reply:
Well, this has been around since forever so I just did not touch it.

Reading it again, I think I'd prefer reusing this menu definition rather than removing it. I'll talk about it during our next grooming.